frontend-react/src/style/annotate.css

```diff
@@ -25,6 +25,7 @@
   border-radius: 5px;
   font-size:20px;
   font-weight: bold;
+
 }
 
 .fileInputButton:focus + .fileLabel {
```

Comment:
I can't add comments for the parts above, but these should be ids and not classes.